app/src/main/java/com/ccc/nameapp/data/source/remote/api/middleware/RxErrorHandlingCallAdapterFactory.kt

```diff
@@ -65,7 +70,8 @@ class RxErrorHandlingCallAdapterFactory private constructor() : CallAdapter.Fact
                     isMaybe -> "Maybe"
                     else -> "Observable"
                 }
-                val errorMessage = "$name return type must be parameterized as $name <Foo> or $name <? extends Foo>"
+                val errorMessage =
+                    "$name return type must be parameterized as $name <Foo> or $name <? extends Foo>"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/c31b8715e13bdfc9754130c3792d53c8f492dbbf/app/src/main/java/com/ccc/nameapp/data/source/remote/api/middleware/RxErrorHandlingCallAdapterFactory.kt#L74"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>